firebase-perf/src/main/java/com/google/firebase/perf/injection/modules/GaugeModule.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 Google LLC
```

Comment:
Usually, it's a good idea to have the Modules close to their dependencies (i.e. in their same package).

Reply:
Do you have any reference for this? I am seeing inappmessaging put all modules together https://github.com/firebase/firebase-android-sdk/tree/master/firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/injection/modules. The reason could be that it is easier to visualize all modules at once.

Reply:
The main reason is because the module is part of the package's functionality. Putting all modules in a shared package means that there is no relation between the module and the classes that need it (other than maybe by name) - it breaks cohesion.

Internally when working with Guice, we always place the module in the same package as the classes it serves. That's the point of the "package" - that it's one "package" with everything necessary to use it.

Reply:
Sounds good to me! Updated.